### ISSUE TITLE
Suggestion: making Alchemy easier to level

### ISSUE THREAD

**Issue by TBlazeWarriorT** [mcMMO-Dev/mcMMO#3178]
Out of all the skills clearly the most repetitive, grindy, boring, painful, slow, hellish one to level up. Legit takes around 600 potions brewn to actually do something. Forces you to craft Thousands of useless potions for hours to unlock something. The only useful reward is Concoctions, but takes a ridiculous amount of time to unlock.
Final suggestion: decrease exp requirement to level up by half. Possibly make it so you get exp from automatic brewer machines. Make it so custom Concoction potions (such as haste and above) give you extra exp (significant amount). Add another reward for levelling alchemy up (e.g making any positive potion effect last 0.1% longer per Alchemy level, decreasing damage taken from poison (having a chance to negate poison damage, maybe))

**Comment by TBlazeWarriorT** [comment]
(don't get offended, but seriously, do you guys play using your own plugin? this skill is a freak to level up, please good God bless you so you fix it on any way)

**Comment by pvprangergod** [comment]
configure it to your liking, thats what config files are for

**Comment by TBlazeWarriorT** [comment]
So it's the server owner's fault? server wasn't mine, I didnt know it. Thanks.
The new stuff that could be added are still a valid suggestion tho

**Comment by pvprangergod** [comment]
yup u can edit mcmmo quite a lot and in terms of balance, entirely. sadly default config is based on a very grindy system

**Comment by Glitchfinder** [comment]
No, you're right. The default alchemy config provides far too little experience per potion, and takes too long to level. For the only skill in the plugin that has settings that *remove* features at low levels, that is absolutely unacceptable.

**Comment by TBlazeWarriorT** [comment]
Thanks a lot for it. Even after one year, I'm glad to know that you agree and will take a look onto it.

**Comment by TBlazeWarriorT** [comment]
After I complained about it, a med-sized server owned asked me if I had a good config file for Alchemy (I don't)... Could you give me some hotfixed values for Alchemy (experience.yml)?
